### PR TITLE
Hosting Metrics: Clone existing log interface into Site Monitoring context

### PR DESCRIPTION
**Pull request by danielbachhuber**
See https://github.com/Automattic/dotcom-forge/issues/3388
See https://github.com/Automattic/dotcom-forge/issues/3393

## Proposed Changes

Introduces 'PHP Logs' and 'Webserver Logs' into the Site Monitoring context:

image

## Testing Instructions

1. Navigate to the existing Site Logs page and verify everything still works as expected: `/site-logs/<site>`
2. Navigate to the Site Monitoring page and verify you can access logs: `/site-monitoring/<site>`
3. Downloading logs should work as expected in both contexts.
4. Filtering by dates should work as expected in both contexts.

### CODE REVIEW

File: client/my-sites/site-monitoring/main.tsx [hunk 1]
```diff
@@ -167,33 +31,14 @@ export function SiteMetrics() {
...
-			</div>
-			<SiteMonitoringBarChart
-				title={ __( 'Requests by HTTP Response Code' ) }
-				{ ...statusCodeRequestsProps }
-			/>
+			<SiteMonitoringTabPanel selectedTab={ page } onSelected={ handleTabSelected }>
+				{ () => (
+					<>
+						{ page === 'metrics' && <MetricsTab /> }
+						{ page !== 'metrics' && <LogsTab logType={ page } /> }
```
Comment (danielbachhuber) [inline client/my-sites/site-monitoring/main.tsx:38]: I'm getting a type warning here but I don't know what it is 😕

Comment (sejas) [reply client/my-sites/site-monitoring/main.tsx:38]: Fixed! https://github.com/Automattic/wp-calypso/pull/80316/commits/76451bae582c5159e2ba6990c9d1c22dd66ac243

There was a bug where `LogsTab` was not restructuring the props object, and it was always showing the Webserver logs and never the PHP logs. Now it's fixed.

**Review by sejas — COMMENTED** [review]
Code looks good. And most of the behaviour works as before.
- Tabs work as expected
- I can download the logs.
- The old `site-logs` page works as before.

I found two main differences:
- SiteMonitoring does not have the Auto-Refresh toggle on the header.
- The pagination is different, and it doesn't work correctly on SiteMonitoring. When I click on a number, `52` , it goes to the next page instead of the page I clicked.

@danielbachhuber , is this something we want to address in this PR?

| | **SiteLogs** | **SiteMonitoring** |
|---|---|---|
| Auto-Refresh | Screenshot 2023-08-08 at 13 53 24 | Screenshot 2023-08-08 at 13 53 04 |
| Pagination | Screenshot 2023-08-08 at 13 52 43 | Screenshot 2023-08-08 at 13 52 52 |


Tests are failing on `client/my-sites/site-monitoring/test/main.js` `useSiteMetrics`.

File: client/my-sites/site-monitoring/main.tsx (hunk 1 above)
Comment (danielbachhuber) [reply client/my-sites/site-monitoring/main.tsx:38]: Thanks!

**Comment by danielbachhuber** [comment]
> * The pagination is different, and it doesn't work correctly on SiteMonitoring. When I click on a number, `52` , it goes to the next page instead of the page I clicked.

I "fixed" this in c510c1fc3a6c20688a0cd1e46e39ada4858c10dd by restoring the CSS that hides the specific pages. Apparently the logging API doesn't support this.

> * SiteMonitoring does not have the Auto-Refresh toggle on the header.

Kicking that conversation to P2 pdKhl6-2cB-p2#comment-3723

**Review by sejas — APPROVED** [review]
@danielbachhuber, Thanks for fixing the pagination.

LGTM!

The autorefresh can be adapted in a following PR.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8692145

Thank you @danielbachhuber for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8692145).

**Pull request merged**